File: 백준/Bronze/17293. 맥주 99병/맥주 99병.c
#include <stdio.h>

int main() {
    int N;
    scanf("%d", &N);

    for (int n = N; n > 0; n--) {
        printf("%d %s of beer on the wall, %d %s of beer.\n", n, (n == 1 ? "bottle" : "bottles"), n, (n == 1 ? "bottle" : "bottles"));
        
        if (n - 1 > 0) {
            printf("Take one down and pass it around, %d %s of beer on the wall.\n\n", n - 1, (n - 1 == 1 ? "bottle" : "bottles"));
        } else {
            printf("Take one down and pass it around, no more bottles of beer on the wall.\n\n");
        }
    }

    printf("No more bottles of beer on the wall, no more bottles of beer.\n");
    printf("Go to the store and buy some more, %d %s of beer on the wall.\n", N, (N == 1 ? "bottle" : "bottles"));
    return 0;
}

File: 백준/Bronze/17293. 맥주 99병/맥주 99병.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        
        for (int n = N; n > 0; n--) {
            String b = (n == 1) ? "bottle" : "bottles";
            String nb = (n - 1 == 1) ? "bottle" : "bottles";
            
            System.out.println(n + " " + b + " of beer on the wall, " + n + " " + b + " of beer.");
            if (n > 1) {
                System.out.println("Take one down and pass it around, " + (n - 1) + " " + nb + " of beer on the wall.");
            } else {
                System.out.println("Take one down and pass it around, no more bottles of beer on the wall.");
            }
            System.out.println();
        }

        System.out.println("No more bottles of beer on the wall, no more bottles of beer.");
        System.out.println("Go to the store and buy some more, " + N + " " + (N == 1 ? "bottle" : "bottles") + " of beer on the wall.");
        scanner.close();
    }
}